_includes/list-partners.html

```diff
@@ -0,0 +1,19 @@
+{% if include.style == "list" %}
+  <ul>
+    {% for agency in project.partner %}
+      {% capture agencyname %}{{ agency | slugify }}{% endcapture %}
+      {% capture partnerurl %}{% for partner in site.data.partners %}{% if partner[0] == agencyname %}{{ partner[1].url }}{% else %} {% endif %}{% endfor %}{% endcapture %}
```

Comment:
I guess I could move the `<ul>` back into the template then just be sure to document that the include only generates the individual list items.